lightningd/peer_control.c

```diff
@@ -1915,6 +1915,8 @@ void channel_watch_wrong_funding(struct lightningd *ld, struct channel *channel)
 void channel_watch_funding(struct lightningd *ld, struct channel *channel)
 {
 	/* FIXME: Remove arg from cb? */
+	log_debug(channel->log, "Watching for funding txid: %s",
+		type_to_string(tmpctx, struct bitcoin_txid, &channel->funding.txid));
 	watch_txid(channel, ld->topology, channel,
```

Comment:
Debug code?

Reply:
I found this *extremely* helpful to understand what was going on during development. I can remove it, but I don't think it's too spammy?